README.md

```diff
@@ -4,8 +4,164 @@ This is a [Singer](https://singer.io) tap that produces JSON-formatted data
 following the [Singer
 spec](https://github.com/singer-io/getting-started/blob/master/SPEC.md).
 
-**TODO** Fill in Readme as development progresses.
+This tap:
 
+- Pulls raw data from the [**CLIENT** API](xxx)
+- Extracts the following resources:
+  - **TBD**
+- Outputs the schema for each resource
+- Incrementally pulls data based on the input state
+
+
+## Streams
+
+[accounts]()
+
+[call_dispositions](https://api.outreach.io/api/v2/docs#callDisposition)
+
+[call_purposes](https://api.outreach.io/api/v2/docs#callPurpose)
+
+[calls](https://api.outreach.io/api/v2/docs#call)
+
+[content_categories](https://api.outreach.io/api/v2/docs#contentCategory)
+
+[duties](https://api.outreach.io/api/v2/docs#duty)
+
+[events](https://api.outreach.io/api/v2/docs#event)
+
+[mailboxes](https://api.outreach.io/api/v2/docs#mailbox)
+
+[mailings](https://api.outreach.io/api/v2/docs#mailing)
+
+[opportunities](https://api.outreach.io/api/v2/docs#opportunity)
+
+[personas](https://api.outreach.io/api/v2/docs#persona)
+
+[prospects](https://api.outreach.io/api/v2/docs#prospect)
+
+[stages](https://api.outreach.io/api/v2/docs#stage)
+
+[tasks](https://api.outreach.io/api/v2/docs#task)
+
+[teams](https://api.outreach.io/api/v2/docs#team)
+
+[users](https://api.outreach.io/api/v2/docs#user)
+
+
+
+
+## Quick Start
+
+1. Install
+
+    Clone this repository, and then install using setup.py. We recommend using a virtualenv:
+
+    ```bash
+    > virtualenv -p python3 venv
+    > source venv/bin/activate
+    > python setup.py install
+    OR
+    > cd .../tap-pinterest-ads
+    > pip install .
+    ```
+2. Dependent libraries
+    The following dependent libraries were installed.
+    ```bash
+    > pip install singer-python
+    > pip install singer-tools
+    > pip install target-stitch
+    > pip install target-json
```

Comment:
This doesn't follow our recommendations of separating the tap virtual environment from the target's

Reply:
that's just saying that they were installed in the vm, i can word it a bit better i guess

Reply:
Step 1 doesn't install all of these things though. And it definitely shouldn't install a target

Reply:
My main concern here is that the tap does not depend on any of these 4 libraries

Reply:
ok, i can take that out